MonkeyLoader.Resonite.Integration/FieldExtensions.cs

```diff
@@ -19,10 +19,10 @@ public static Action<IChangeable> SyncWithConfigKey<T>(this IField<T> field, IDe
             void FieldChangedHandler(IChangeable _)
             {
                 if (field.Value.Equals(configKey.GetValue())) return;
-                if (allowWriteBack)
-                    configKey.SetValue(field.Value, eventLabel);
-                else
+                if (!(allowWriteBack && configKey.TrySetValue(field.Value, eventLabel)))
```

Comment:
I think this conditional may be easier to understand if it was `if (!allowWriteBack || !configKey.TrySetValue(...))`

Reply:
Then it would try to set the config key even if writeBack is false, which is not supposed to happen